### PR TITLE
[13.0][MIG] account_reconcile_payment_order: Migration to 13.0

### PR DESCRIPTION
**Pull request by joao-p-marques**
Supersedes https://github.com/OCA/account-reconcile/pull/358 to fix tests.

@Tecnativa 
TT27489

ping @pedrobaeza @victoralmau

### CODE REVIEW

File: account_reconcile_payment_order/i18n/de.po [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+
+#. module: account_reconcile_payment_order
+#: model:ir.model,name:account_reconcile_payment_order.model_account_reconciliation_widget
+msgid "Account Reconciliation widget"
+msgstr ""
+
+#~ msgid "Bank Statement Line"
+#~ msgstr "Kontoauszugzeile"
```
Comment (victoralmau) [inline account_reconcile_payment_order/i18n/de.po]:
```suggestion
```

File: account_reconcile_payment_order/i18n/fr.po [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: nplurals=2; plural=(n > 1);\n"
+
+#. module: account_reconcile_payment_order
+#: model:ir.model,name:account_reconcile_payment_order.model_account_reconciliation_widget
+msgid "Account Reconciliation widget"
+msgstr ""
+
+#~ msgid "Bank Statement Line"
+#~ msgstr "Ligne de relevé bancaire"
```
Comment (victoralmau) [inline account_reconcile_payment_order/i18n/fr.po]:
```suggestion
```

File: account_reconcile_payment_order/i18n/it.po [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+
+#. module: account_reconcile_payment_order
+#: model:ir.model,name:account_reconcile_payment_order.model_account_reconciliation_widget
+msgid "Account Reconciliation widget"
+msgstr ""
+
+#~ msgid "Bank Statement Line"
+#~ msgstr "Linea estratto conto"
```
Comment (victoralmau) [inline account_reconcile_payment_order/i18n/it.po]:
```suggestion
```

File: account_reconcile_payment_order/i18n/nl_NL.po [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+
+#. module: account_reconcile_payment_order
+#: model:ir.model,name:account_reconcile_payment_order.model_account_reconciliation_widget
+msgid "Account Reconciliation widget"
+msgstr ""
+
+#~ msgid "Bank Statement Line"
+#~ msgstr "Bankafschriftregel"
```
Comment (victoralmau) [inline account_reconcile_payment_order/i18n/nl_NL.po]:
```suggestion
```

File: account_reconcile_payment_order/i18n/sl.po [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+"Plural-Forms: nplurals=4; plural=(n%100==1 ? 0 : n%100==2 ? 1 : n%100==3 || n"
+"%100==4 ? 2 : 3);\n"
+
+#. module: account_reconcile_payment_order
+#: model:ir.model,name:account_reconcile_payment_order.model_account_reconciliation_widget
+msgid "Account Reconciliation widget"
+msgstr ""
+
+#~ msgid "Bank Statement Line"
+#~ msgstr "Postavka bančnega izpiska"
```
Comment (victoralmau) [inline account_reconcile_payment_order/i18n/sl.po]:
```suggestion
```

File: account_reconcile_payment_order/tests/test_account_reconcile_payment_order.py [hunk 6]
```diff
@@ -0,0 +1,113 @@
...
+        self.assertEqual(len(self.inbound_order.payment_line_ids), 2)
+        self.inbound_mode.write(
+            {"offsetting_account": "bank_account", "move_option": "line"}
+        )
+        # Prepare payment order
+        self.inbound_order.draft2open()
+        self.inbound_order.open2generated()
+        self.inbound_order.generated2uploaded()
+        # Check widget result
+        res = self.widget_obj.get_bank_statement_line_data(self.statement.line_ids.ids,)
```
Comment (victoralmau) [inline account_reconcile_payment_order/tests/test_account_reconcile_payment_order.py:59]:
```suggestion
        res = self.widget_obj.get_bank_statement_line_data(self.statement.line_ids.ids)
```

File: account_reconcile_payment_order/tests/test_account_reconcile_payment_order.py [hunk 7]
```diff
@@ -0,0 +1,113 @@
...
+                "move_option": "line",
+            }
+        )
+        self.assertEqual(len(self.inbound_order.payment_line_ids), 2)
+        # Prepare payment order
+        self.inbound_order.draft2open()
+        self.inbound_order.open2generated()
+        self.inbound_order.generated2uploaded()
+        # Check widget result
+        res = self.widget_obj.get_bank_statement_line_data(self.statement.line_ids.ids,)
```
Comment (victoralmau) [inline account_reconcile_payment_order/tests/test_account_reconcile_payment_order.py]:
```suggestion
        res = self.widget_obj.get_bank_statement_line_data(self.statement.line_ids.ids)
```

**Review by victoralmau — APPROVED** [review]

**Comment by joao-p-marques** [comment]
@pedrobaeza about the translations suggestions, does @oca-transbot do these changes? Or should I do them manually?

**Comment by pedrobaeza** [comment]
OCA bot doesn't purge deprecated translations. It's not required to purge them though.

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**